src/_locales/fa/messages.json

```diff
@@ -0,0 +1,431 @@
+    }
+  },
+  "optionQrQuietZoneDescr": {
+    "message": "The number of border modules around the QR code colored in the background color.",    
+    "description": "The description of the quiet zone option."
+  },
+  "optionQrQuietZoneDescrLink": {
+    "message": "https://github.com/rugk/offline-qr-code/wiki/FAQ#what-is-a-quiet-zone",
+    "description": "The 'Learn more' link in the description of the quiet zone option."
+  },
+
+  "optionQrCodeColor": {
+    "message": "رنگ کد کیو ار:",
+    "description": "This is an option shown in the add-on settings."
+  },
+
+  "optionQrCodeBackgroundColor": {
+    "message": "رنگ پس‌زمینه کد کیو ار:",
+    "description": "This is an option shown in the add-on settings."
+  },
+
+  "optionErrorCorrection": {
+    "message": "سطح تصحیح خطا:",
+    "description": "Error correction setting for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction."
+  },
+  "optionEcLow": {
+    "message": "Low (7%)",
+    "message": "کم (۷ درصد)",
+    "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction."
+  },
+  "optionEcMedium": {
+    "message": "متوسط (۱۵ درصد)",
+    "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction."
+  },
+  "optionEcQuartile": {
+    "message": "یک چهارم (۲۵ درصد)",
+    "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction."
+  },
+  "optionEcHigh": {
+    "message": "زیاد (۳۰ درصد)",
+    "description": "Option of error correction level for QR code. See https://en.wikipedia.org/wiki/QR_code#Error_correction."
+  },
+  "optionErrorCorrectionDescr": {    
+    "message": "این مقداد نشان می دهد که چه مقدار از کد می توانید غیر قابل خواندن باشد در حالی که همچنان میتوان کد را اسکن کرد.",
+    "description": "The description of the error correction option."
+  },
+  "optionErrorCorrectionDescrLink": {
+    "message": "https://en.wikipedia.org/wiki/QR_code#Error_correction",
+    "description": "The 'Learn more' link in the description of the error correction option."
+  },
+
+  "optionAutoGetSelectedText": {    
+    "message": "استفاده خودکار از متن انتخاب شده داخل وبسایت",
+    "description": "This is an option shown in the add-on settings."
+  },
+  "optionUseMonospaceFont": {
+    "message": "استفاده از فونت monospace",
+    "description": "This is an option shown in the add-on settings."
+  },
+
+  "optionDebugMode": {
+    "message": "فعال کردن حالت عیب‌یابی",
+    "description": "This is an option shown in the add-on settings."
+  },
+  "optionDebugModeDescr": {    
+    "message": "این گزینه برای دریافت اطلاعات بیشتر از جنس لاگ میباشد(برای گزارش و پیدا کردن خطا استفاده می شود)",
+    "description": "The description shown for the debug mode (optionDebugMode)."
+  },
+
+  "translatorCredit": {
+    "message": "This add-on has been translated into English by $TRANSLATORS$.",
```

Comment:
This message also has to be translated. And say "Persian" of course, not English.

See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translations for how these placeholders are translated, especially [the official doc](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales).

Reply:
Ah, actually I see you've duplicated the line, that seems to be the error here.